pkgs/servers/web-apps/wordpress/default.nix

```diff
@@ -12,17 +19,23 @@ stdenv.mkDerivation rec {
   installPhase = ''
     mkdir -p $out/share/wordpress
     cp -r . $out/share/wordpress
+    rm -r $out/share/wordpress/wp-content/plugins/*
+    rm -r $out/share/wordpress/wp-content/themes/*
+    ${lib.concatMapStringsSep "\n" (theme: "cp -r ${theme} $out/share/wordpress/wp-content/themes/${theme.passthru.wpName or theme.pname}") wpThemes}
```

Comment:
It's been a (short) while but I think I'm currently breaking theme config for the built-in themes. maybe set wpThemes to null by default and only do these four lines here if it's not the default. Then users won't have the benefit of this when they're just using the wordpress package but we can at least hide this **really important** bugfix here without breaking existing users on accident. 

The bugfix is that the directory name doesn't contain the version as some themes/plugins are stupid enough to use that to find their config in the db.

Reply:
The upstream package actually does this right so this should work.

Diff:
```
nix build --out-link original github:NixOS/nixpkgs#wordpress
nix build --out-link modified .#wordpress
diff --color -r original modified
```
looks good.